website/api/zkvm/precompiles.md

```diff
@@ -12,52 +12,69 @@ with significantly less resources [^1].
 
-### Using Accelerated Crates
+### ECDSA
 
-When using any of the crates listed above directly, specifying the dependency as
-a [git dependency][git-dep]. For example:
+| Crate | Versions supported | Patch Statement Example |
+|-------|-------------------|------------------------|
+| [`k256`](https://github.com/risc0/RustCrypto-elliptic-curves/releases) | 0.13.4, 0.13.3, 0.13.2, 0.13.1 | `k256 = { git = "https://github.com/risc0/RustCrypto-elliptic-curves", tag = "k256/v0.13.3-risczero.1" }` [^2] |
 
-```toml
-[dependencies.sha2]
-git = "https://github.com/risc0/RustCrypto-hashes"
-tag = "sha2-v0.10.6-risczero.0"
-```
+### EDDSA
+
+| Crate | Versions supported | Patch Statement Example |
+|-------|-------------------|------------------------|
+| [`curve25519-dalek`](https://github.com/risc0/curve25519-dalek/releases) | 4.1.2, 4.1.1, 4.1.0 | `ed25519-dalek = { git = "https://github.com/risc0/ed25519-dalek", tag = "curve25519-4.1.2-risczero.0" }` |
+
+### RSA
+
+| Crate | Versions supported | Patch Statement Example |
+|-------|-------------------|------------------------|
+| [`rsa`](https://github.com/risc0/RustCrypto-RSA/releases) | 0.9.6 | `rsa = { git = "https://github.com/risc0/RustCrypto-RSA", tag = "v0.9.6-risczero.0" }` [^2] |
+
+### Other Accelerated Crates
+
+| Crate | Versions supported | Patch Statement Example |
+|-------|-------------------|------------------------|
+| [`crypto-bigint`](https://github.com/risc0/RustCrypto-crypto-bigint/releases) | 0.5.5, 0.5.4, 0.5.3, 0.5.2 | `crypto-bigint = { git = "https://github.com/risc0/RustCrypto-crypto-bigint", tag = "v0.5.5-risczero.0" }` |
+
+Make sure that your dependency gives the same patch version of the crate as listed in
+the git tag of the patch. If you need other patch versions or crates than listed here, please reach
+out to us on [Discord][discord-url] or otherwise!
+
+If using `tag = "sha2-v0.10.8-risczero.0"`, the dependency should be:
 
-Make sure that your dependency gives the same version of the crate as listed in
-the patch tag. In this example, the dependency would be
 ```toml
 [dependencies]
-sha2 = "=0.10.6"
+sha2 = "=0.10.8"
+
+[patch.crates-io]
+sha2 = { git = "https://github.com/risc0/RustCrypto-hashes", tag = "sha2-v0.10.8-risczero.0" }
 ```
 
 In some situations, for example when a patch is used indirectly, you may
-need to update the version specifically to update your lockfile. For example:
-```toml
-cargo update -p sha2 --precise 0.10.6
+need to update the version specifically to update your lockfile for your guest.
+For example:
+
+```sh
+cargo update -p sha2 --precise 0.10.8
 ```
 
+> Note: To ensure that the patch is being applied, search for the crate in the `Cargo.lock` file
+> of your guest to ensure that it references the fork repository. It is generally good practice to
+> commit the `Cargo.lock` file to git, to ensure versions don't get accidentally updated.
```

Comment:
Maybe link https://blog.rust-lang.org/2023/08/29/committing-lockfiles.html for this as well?